lib/localization/localization.dart

```diff
@@ -65,8 +65,6 @@ extension AppLocalizationsExtension on AppLocalizations {
             return artist;
           case SongSortFeature.album:
             return album;
-          default:
```

Comment:
Since the `*SortFeature`s are now regular enums, we can exhaustively match on them. This means that if we add a new `SongSortFeature` we won't forget to add another case here because we'll be notified at compiletime, instead of runtime.